Doc/library/fnmatch.rst

```diff
@@ -35,9 +35,9 @@ For example, ``'[?]'`` matches the character ``'?'``.
 
 Note that the filename separator (``'/'`` on Unix) is *not* special to this
 module.  See module :mod:`glob` for pathname expansion (:mod:`glob` uses
-:func:`fnmatch` to match pathname segments).  Similarly, filenames starting with
-a period are not special for this module, and are matched by the ``*`` and ``?``
-patterns.
+:func:`~fnmatch.filter` to match pathname segments).  Similarly, filenames
```

Comment:
Why not just ``` :func:`.filter` ```?

Reply:
I didn't know about that shortcut :) Applied!

Reply:
Does it work? I didn't check it.

Reply:
Yeap, I checked it before doing the commit.

Reply:
Yes, it does. Just checked it locally. Thanks for the tip!

![screen shot 2018-11-07 at 11 37 36 pm](https://user-images.githubusercontent.com/3972343/48151081-6c2c5400-e2e6-11e8-9eeb-c0d35e1f1638.png)